advanced-cache.php

```diff
@@ -338,14 +323,6 @@ function do_variants($dimensions = false) {
 		else
 			wp_cache_set("{$this->url_key}_vary", $dimensions, $this->group, $this->max_age + 10);
```

Comment:
I don't think there's a reason to keep any of this code around since we're removing all the functionality around varying on functions. I don't think we need to be calling `do_variants()` at all at this point, we already know `$batcache->vary` is an empty array and trust that will never change. Maybe remove `$batcache->vary` altogether since it's not used outside of this context.

Varying otherwise is done via `$batcache->unique` as far as I know.